src/sentry/tasks/delete_seer_grouping_records.py

```diff
@@ -56,7 +57,13 @@ def call_delete_seer_grouping_records_by_hash(
         project = group.project if group else None
     if (
         project
-        and features.has("projects:similarity-embeddings-grouping", project)
+        and (
+            features.has("projects:similarity-embeddings-grouping", project)
+            or (
+                project.get_option("sentry:similarity_backfill_completed")
+                and randint(1, 100) <= options.get("similarity.delete_task_EA_rollout_percentage")
```

Comment:
Just to confirm, we don't want any stickyness on the project/org and it's ok if it's a randomly distributed percentage

Reply:
Correct, we just want to make sure our task and seer can handle the load slowly. I don't think this slower rollout is strictly necessary, but wanted to be cautious.